MozillaFoundation/foundation.mozilla.org
# Page types report

# Description



Add a Page types report.

Covers tickets #10853 and #10854.

Link to sample test page: "cms/reports/page-types-report/"

To test:

1. Login to admin
2. Go to reports
3. Go to "Page Types"
4. See report

# Checklist



**Tests**

**Changes in Models:**

**Documentation:**
